srcpkgs/typst/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'typst'
+pkgname=typst
+version=23.03.28
+revision=1
+build_style=cargo
+configure_args="-p typst-cli"
```

Comment:
There's a fix on upstream that allows installing with `--path cli`: https://github.com/typst/typst/commit/613c644f0479b2aae6be96fa389352883a2321c3

Reply:
thanks